src/components/Sidebar/Sidebar.jsx

```diff
@@ -1,28 +1,42 @@
-import { AiOutlineHome,MdOutlineExplore ,MdOutlineFeaturedPlayList,ImClock} from "../../icons/icons";
+import { AiOutlineHome,MdOutlineExplore,FaHistory ,MdOutlineFeaturedPlayList,ImClock} from "../../icons/icons";
 import "./Sidebar.css"
+import {useNavigate} from "react-router-dom";
+const sidebarData=[
+  {
+    id:1,
+    navIcon: <AiOutlineHome className="icon size-xs" />,
+    navTxt:"Home",
+    navLink:"/",
+  },
+  {
+    id:2,
+    navIcon: <MdOutlineExplore className="icon size-xs" />,
+    navTxt:"Explore",
+    navLink:"/explore",
+  },
+  {
+    id:3,
+    navIcon: <MdOutlineFeaturedPlayList className="icon size-xs" />,
+    navTxt:"Playlist",
+    navLink:"/playlist",
+  },
+  {
+    id:5,
+    navIcon: <FaHistory className="icon size-xs clock-icon" />,
+    navTxt:"History",
+    navLink:"/history",
+  },
+]
 function Sidebar() {
+  const navigate = useNavigate();
   return (
-    <>
     <div className="flex sidebar-container">
-      <div className="flex sidebar-icon-container">
-        <AiOutlineHome className="icon size-xs" />
-        <span className="lt-bold nav-icon-txt" >Home</span>
-      </div>
-      <div className="flex sidebar-icon-container">
-          <MdOutlineExplore className="icon size-xs"/>
-          <span className="lt-bold nav-icon-txt" >Explore</span>
-      </div>
-      <div className="flex sidebar-icon-container">
-          <MdOutlineFeaturedPlayList className="icon size-xs"/>
-          <span className="lt-bold nav-icon-txt" >Playlist</span>
-      </div>
-      <div className="flex sidebar-icon-container">
-          <ImClock className="icon size-xs history-icon  "/>
-          <span className="lt-bold nav-icon-txt" >History</span>
-      </div>
+    {sidebarData.map(sidebarItem=>(
+    <div onClick={()=>navigate(sidebarItem.navLink)} key={sidebarItem.id} className="flex sidebar-icon-container">
+        {sidebarItem.navIcon}
```

Comment:
Try destructing here. It will look cleaner

Reply:
Made the changes.